src/com/lftechnology/jan12/assignment/LinkedListManipulation.java

```diff
@@ -0,0 +1,53 @@
+package com.lftechnology.jan12.assignment;
+
+import java.util.Arrays;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * <ul>
+ * <li>a. Create a linked list named one and two.</li>
+ * <li>b. Add some elements to both one (one, two, three, four, five) and two (six, seven, eight, nine, and ten).</li>
+ * <li>c. Merge the words from two into one.</li>
+ * <li>d. Remove every second word from two.</li>
+ * <li>e. Remove all words in two from one.</li>
+ * <li>f. Print the value after each operation.</li>
+ * </ul>
+ * 
+ * @author bibhushan<bibhushanjoshi@lftechnology.com>
+ * 
+ */
+public class LinkedListManipulation {
+	private static final Logger LOGGER = Logger.getLogger(LinkedListManipulation.class.getName());
+
+	public static void main(String[] args) {
+		List<String> one = new LinkedList<String>(Arrays.asList("one", "two", "three", "four", "five"));
+		List<String> two = new LinkedList<String>(Arrays.asList("six", "seven", "eight", "nine", "ten"));
+		LOGGER.log(Level.INFO, "Given Lists are:\n {0} \n {1}", new Object[] { one, two });
+		// adds the given two linked list
+		one.addAll(two);
+		LOGGER.log(Level.INFO, "After two list added:" + one);
+		// removes every second element from linked list
+		removeSecond(one);
+		LOGGER.log(Level.INFO, "After second element removed: " + one);
+		// removes every remaining element of linked list two(second list)
+		one.removeAll(two);
+		LOGGER.log(Level.INFO, "After all second list element removed: " + one);
+	}
+
+	/**
+	 * This method takes in {@link LinkedList} and removes every second element from the list
+	 * 
+	 * @param inputList
+	 *            {@link LinkedList} from which every second element is to be removed
+	 * 
+	 * @author bibhushan<bibhushanjoshi@lftechnology.com>
+	 */
+	public static void removeSecond(List<String> inputList) {
+		for (int i = 0; i < inputList.size(); i++) {
```

Comment:
@Msanjib the size of the inputList is changing in every iteration so i guess its inputList.size() is required inside the loop. If the variable for storing it is declared outside the loop then it is required to update it manually within the loop.